pyproject.toml

```diff
@@ -99,8 +99,12 @@ version_scheme = "pep440"
 [tool.poetry]
 packages = [{ include = "commitizen" }, { include = "commitizen/py.typed" }]
 
+[tool.poetry.requires-plugins]
+"poethepoet" = ">=0.32.2"
```

Comment:
The last time I tried this when testing project template generation, it worked weirdly. I probably should try it again. maybe it's fixed